sale_stock_expiry_date_on_qty_at_date_widget/README.rst

```diff
@@ -0,0 +1,122 @@
+============================================
+Sale Stock Expiry Date on Qty at Date widget
+============================================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:a3131c0f8cd0f154875027d0a104e1e3c720c1d462b9c34c4b186a808fedfbc9
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Alpha-red.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Alpha
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--workflow-lightgray.png?logo=github
+    :target: https://github.com/OCA/sale-workflow/tree/16.0/sale_stock_expiry_date_on_qty_at_date_widget
+    :alt: OCA/sale-workflow
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/sale-workflow-16-0/sale-workflow-16-0-sale_stock_expiry_date_on_qty_at_date_widget
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/builds?repo=OCA/sale-workflow&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+This module extends the functionality of Qty at Date Widget to display
+next Expiry Date and to allow you to inform your customers properly
+about the goods they are going to purchase.
+
+.. IMPORTANT::
+   This is an alpha version, the data model and design can change at any time without warning.
+   Only for development or testing purpose, do not use in production.
+   `More details on development status <https://odoo-community.org/page/development-status>`_
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Use Cases / Context
+===================
+
+This module was developed because sometimes Commercial users want to
+know where is the Expiraiton Date of the products.
+
+It will be useful for you if your Commercial users need to share this
+informations with its customers.
+
+Usage
+=====
+
+To use this module, you need to:
```

Comment:
I have to install sale_management because in runboat it wasn't installed. So I guess there is not dependency 😄 just a comment. Maybe nothing to FIX